### PR TITLE
Pop up modal & mobile menu

### PR DESCRIPTION
**Pull request by sergiogval**
On this branch, I did a pop up modal according with template. Also I improved some features on  my Mobile Menu.

### CODE REVIEW

**Review by DanteAlonsoHT — COMMENTED** [review]
<h1>Status: Invalidating project 🚫 </h1>

Hi @sergiogval, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

## You have submitted a project, that doesn't pass the linters and also you give me the URL to go to your repository instead of the URL to go to your pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._ 😉

**Comment by sergiogval** [comment]
> 
> # Status: Invalidating project 🚫
> 
> Hi @sergiogval,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> ## Invalid Code Review Request
> ## You have submitted a project, that doesn't pass the linters and also you give me the URL to go to your repository instead of the URL to go to your pull request.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **Does not count** into the code reviews limit._ 😉



> e URL to go to your repository instead of the URL to go to your pull request.

As far as I know, the instruction is to share the repo link, not the pull request one. You can read this guideline when submitting a CR. However, This time I sent the link you asked for.

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @sergiogval, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/emailvalid.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+var email = document.getElementById('email');
+email.setAttribute('pattern', '[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,3}$');
+document.body.appendChild(email);
```
Comment (mke2111) [inline js/emailvalid.js:3]: - [ ] An error message should show up close to the "submit" button. Your page is currently missing this feature.

Comment (sergiogval) [reply js/emailvalid.js:3]: This is not in the scope of this project US. But I will take your advice for the proper project.

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @sergiogval, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,355 +1,302 @@
...
+        </div>    
+      </div>
+      <div class="card text-center col-sm-3">
+        <img src="./images/Untitled.png" class="card-img-top" alt="placeholder" />
+        <div class="card-body">
+          <h5 class="card-title">Multi-post Stories Gain+Glory</h5>
+          <p class="card-text">
+            <span class="badge rounded-pill bg-light text-dark">Ruby on Rails</span>
+            <span class="badge rounded-pill bg-light text-dark">CSS</span>
+            <span class="badge rounded-pill bg-light text-dark">JavaScript</span>
```
Comment (mke2111) [inline index.html:149]: - [ ] For this milestone, you are required to remove all HTML structure for all projects from the HTML file. You are instead required to use javascript code to implement an array of objects to store JavaScript objects containing the information for each project.
- [ ] Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]

```

File: js/modal.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+const wahidullah = document.getElementById('btnShow');
+const sergio = document.getElementById('btnClose');
+
+wahidullah.addEventListener('click', () => {
+    document.querySelector('.modal').style.display = 'block';
+});
+
+sergio.addEventListener('click', () => {
+    document.querySelector('.modal').style.display = 'none';
+});
```
Comment (mke2111) [inline js/modal.js:10]: - [ ] Add the Javascript code in this file, or at least in the `main.js` file

File: index.html [hunk 4]
```diff
@@ -1,355 +1,302 @@
...
+        
+        <div class="modal-body row">
+          <div class="col">
+          <h2 class="modal-title" >Keeping track of thousand of components</h2>
+            <span class="badge rounded-pill bg-light text-dark">CodeKit</span>
+            <span class="badge rounded-pill bg-light text-dark">GitHub</span>
+            <span class="badge rounded-pill bg-light text-dark">JavaScript</span>
+            <span class="badge rounded-pill bg-light text-dark">BootStrap</span>
+            <span class="badge rounded-pill bg-light text-dark">Terminal</span>
+            <span class="badge rounded-pill bg-light text-dark">Codepen</span>
```
Comment (mke2111) [inline index.html:48]: - [ ] The popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colors, images, tests, margins).

File: js/emailvalid.js (hunk 1 above)
Comment (mke2111) [reply js/emailvalid.js:3]: No worries, just make sure you implement all these features for the Validate contact form milestone.

**Pull request closed without merging**